### DISCUSSION TITLE
Best App - Read Feature

### DISCUSSION THREAD

**Discussion by txprojects88** [aeharding/voyager#131] [Ideas]
Hi, first off thanks for creating such a wonderful app. Closest to Apollo I have seen. Can you add the feature where a post doesn't display again if you have already scrolled through the post or read it? 

Thanks again!

**Comment by aeharding** [comment]
Hi thanks! Yes, this is planned, but dependent on https://github.com/aeharding/wefwef/pull/63

**Comment by txprojects88** [comment]
Awesome. Thanks!
